docs/reference/esql/functions/kibana/inline_cast.json

```diff
@@ -0,0 +1,19 @@
+{
+  "bool" : "boolean",
```

Comment:
Oooo I had missed this one 🙇

Reply:
I figured it was best if I just scraped it from the map we really use. I wasn't sure what the list is!

Reply:
I really like the way you're tying the metadata we get to actual code and cases in Elasticsearch 👍